databricks_cli/click_types.py

```diff
@@ -104,6 +106,29 @@ def __init__(self):
     def set_debug(self, debug=False):
         self._debug = debug
 
+        if not self._debug:
+            return
+
+        # These two lines enable debugging at httplib level (requests->urllib3->http.client)
+        # You will see the REQUEST, including HEADERS and DATA,
+        # and RESPONSE with HEADERS but without DATA.
+        # The only thing missing will be the response.body which is not logged.
+        try:
+            import http.client as http_client
+        except ImportError:
+            # Python 2
+            import httplib as http_client
+
+        click.echo("HTTP debugging enabled")
+        http_client.HTTPConnection.debuglevel = 1
+
+        # You must initialize logging, otherwise you'll not see debug output.
+        logging.basicConfig()
```

Comment:
Why do we need this line? Looks like they will go to stderr by default?
> By default, no destination is set for any logging messages. You can specify a destination (such as console or file) by using basicConfig() as in the tutorial examples. If you call the functions debug(), info(), warning(), error() and critical(), they will check to see if no destination is set; and if one is not set, they will set a destination of the console (sys.stderr) and a default format for the displayed message before delegating to the root logger to do the actual message output.

Reply:
In fact, why do we need anything to do with requests.packages.urllib3? I removed all of that and it still worked.

Reply:
we might not need the request_log lines, lemme go back and check.  It's possible that since the patch was written, things have changed enough these lines are no longer needed.

Reply:
Fixed.